roles/aap_setup_install/tasks/main.yml

```
@@ -18,11 +18,11 @@
 
 - name: Check Automation Hub Running
   uri:
-    url: "https://{{ ah_hostname }}/api/galaxy/"
+    url: "https://{{ ah_hostname | default((aap_setup_prep_inv_nodes['automationhub'].keys() | list)[0]) }}/ui/"
```

Again, I think this should probably be the api endpoint?
```suggestion
    url: "https://{{ ah_hostname | default((aap_setup_prep_inv_nodes['automationhub'].keys() | list)[0]) }}/api/galaxy/"
```

---

Good catch

---

Fixed and updated